packages/core/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, h, Host, Prop } from "@stencil/core";
+
+@Component({
+  tag: "fl-spinner",
+  styleUrl: "spinner.scss",
+  shadow: true,
+})
+export class Spinner {
+  @Prop() size: "sm" | "md" | "lg" = "md";
```

Comment:
I think we should change these to `small`, `medium`, and `large` because the purpose is more clear